solr/solrj/src/java/org/apache/solr/client/solrj/impl/CloudLegacySolrClient.java

```diff
@@ -238,16 +238,6 @@ public Builder canUseZkACLs(boolean canUseZkACLs) {
       return this;
     }
 
-    /**
-     * Sets a default collection for all collection-based requests.
-     *
-     * <p>Identical to {@link #withDefaultDataStore(String)} for this builder
-     */
-    public Builder withDefaultCollection(String defaultCollection) {
-      this.defaultDataStore = defaultCollection;
-      return this;
-    }
-
```

Comment:
why?

Reply:
TL;DR - this code is moved, not deleted.

1. Cloud Clients have long had a builder method called `withDefaultCollection` for several years now, pre-existing any of the recent changes around this.
2. PR#2066 added `withDefaultDataStore` (now `withDefaultCollection`) for all clients.
3. Since all clients builders now have this method, it can live in the SolrClientBuilder parent class instead of only in CloudLegacySolrClient.